packages/binding/src/transformer.rs

```diff
@@ -669,6 +663,54 @@ impl Transformer {
     }
   }
 
+  #[napi]
+  /// Support CSS3 color, e.g. rgba(255, 255, 255, .8)
+  pub fn from_svg(
+    input: Either<String, Buffer>,
+    background: Option<String>,
+  ) -> Result<Transformer> {
+    let mut fontdb = Database::new();
+    fontdb.load_system_fonts();
```

Comment:
Can we consider providing an API to load only specified fonts in the future? This would improve the speed.

Reply:
Good Idea! Welcome to PR~ 🙏